Tests/CSharp/ExpressionTests.cs

```diff
@@ -633,5 +633,62 @@ public string TestMethod()
     }
 }");
         }
+
+
+
+
+
+        [Fact]
+        public void IntegerDivideExpression()
+        {
+            TestConversionVisualBasicToCSharp(@"Class TestClass
+    Sub TestMethod(ByRef i As Integer)
+        Dim k as Integer
+        k = i \ 1000000
+    End Sub
+End Class"
+, @"class TestClass
+{
+    public void TestMethod(ref int i)
+    {
+        int k;
+        k = i / 1000000;
+    }
+}");
+        }
+
+
+
+
+        [Fact]
+        public void NarrowingWideningExpression()
+        {
+            //TestConversionVisualBasicToCSharp => Failing! because of cr/lf after "operator MyInt" in expected CS below:
+            TestConversionVisualBasicToCSharpWithoutComments(@"Public Class MyInt
+    Public Shared Narrowing Operator CType(i As Integer) As MyInt
+            Return New MyInt()
+        End Operator
+        Public Shared Widening Operator CType(myInt As MyInt) As Integer
+            Return 1
+        End Operator
+    End Class"
+, @"public class MyInt
+{
+    public static explicit operator MyInt
+(int i)
+    {
+        return new MyInt();
+    }
+    public static implicit operator int
+(MyInt myInt)
+    {
+        return 1;
+    }
+}");
+        }
```

Comment:
Yep you're on the right track!

Reply:
Here there's an issue I didn't knew how to fix: the parameters list is send to the next line instead of staying on the same line as the operator declaration.
It seems it's roslyn which generate this line feed at
*resultNode.ToFullString()*
# projectconvertion.cs line 61
I tried to find any option to prevent this to happen but didn't find any. Maybe you already know the answer. Indeed I had to use the TestConversionVisualBasicToCSharpWithoutComments instead of the TestConversionVisualBasicToCSharp which is clearly failing because of this evil line feed :-).

Reply:
Yes I'm happy with that fix for the test failure. Good job figuring it out - I've added this as a separate issue to try to make it clearer for others (#45).

Reply:
I think this test probably belongs in `MemberTests` since it's really about a special kind of class member. Would you mind moving it into there?

Reply:
Will do...if you can wait...I'm on the steep slope of the learning curve ;-)

Reply:
@BaronBodissey No worries - I'll tidy things up and get it merged now